src/lib.rs

```diff
@@ -305,12 +299,12 @@ impl Default for PanCam {
             grab_buttons: vec![MouseButton::Left, MouseButton::Right, MouseButton::Middle],
             enabled: true,
             zoom_to_cursor: true,
-            min_scale: 0.00001,
-            max_scale: None,
-            min_x: None,
-            max_x: None,
-            min_y: None,
-            max_y: None,
+            min_scale: f32::NEG_INFINITY,
```

Comment:
Was this an accidental change? How does a negative/zero scale behave?

Reply:
Oops, you're right. I didn't test this properly😳. I've reverted the change